pkg/models/storage/storage.go

```diff
@@ -27,6 +28,11 @@ import (
 	"kubesphere.io/kubesphere/pkg/informers"
 )
 
+const (
+	IsDefaultStorageClassAnnotation     = "storageclass.kubernetes.io/is-default-class"
+	BetaIsDefaultStorageClassAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
+)
+
 type ScMetrics struct {
```

Comment:
same here, unnecessary exported

Reply:
remember to remove unused function `init()` down below

Reply:
> remember to remove unused function `init()` down below

Sure